monitor-server/src/main.py

```diff
@@ -0,0 +1,35 @@
+from http.client import HTTPException
+from fastapi import FastAPI
+import uvicorn
+
+app = FastAPI()
+
+
+emails = [{
+        "topic": "jobs emails",
+        "address" : "email address",
```

Comment:
why not using correct dummy mail address? like test@test.de

Reply:
ok, will do that...